### PR TITLE
style: update search suggestion

### LINKED ISSUE #7078: Search trending alignment
image
- the top part should look more like this: 
- 
image
- wrapped in container with 32px left right padding 24px bottom top

image
- Please center out the text with the preview img on the y-axis 

- the divider should not be full-width 
- should fill out the container till it hit the padding 
image

### PR DESCRIPTION
**Pull request by floyd-li**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #7078
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [ ] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at c6d3022</samp>

Improved UI and UX of `SearchSuggestion.vue` component. Hid empty search history, fixed trending tab layout, and added vertical space for search result items.

### <samp>🤖 Generated by Copilot at c6d3022</samp>

> _The search suggestion component_
> _Got a UI improvement_
> _No more empty div_
> _Better padding and alignment_
> _And some vertical space for each element_

### CODE REVIEW

File: components/search/SearchSuggestion.vue [hunk 1]
```diff
@@ -196,8 +196,16 @@
...
       </div>
     </div>
-    <NeoTabs v-show="!name" v-model="activeTrendingTab" expanded>
-      <NeoTabItem label="Trending" value="Trending">
+    <NeoTabs
+      v-show="!name"
+      v-model="activeTrendingTab"
+      expanded
+      nav-tabs-class="pt-2 pl-6 pr-6"
+      content-class="px-0 py-4">
```
Comment (Jarsen136) [inline components/search/SearchSuggestion.vue:204]:
```suggestion
      content-class="p-4">
```

File: components/search/SearchSuggestion.vue [hunk 2]
```diff
@@ -196,8 +196,16 @@
         </div>
       </div>
     </div>
-    <NeoTabs v-show="!name" v-model="activeTrendingTab" expanded>
-      <NeoTabItem label="Trending" value="Trending">
+    <NeoTabs
+      v-show="!name"
+      v-model="activeTrendingTab"
+      expanded
+      nav-tabs-class="pt-2 pl-6 pr-6"
```
Comment (Jarsen136) [inline components/search/SearchSuggestion.vue:203]:
```suggestion
      nav-tabs-class="pt-2 px-6"
```

File: components/search/SearchSuggestion.vue [hunk 3]
```diff
@@ -170,7 +170,7 @@
         </template>
       </NeoTabItem>
     </NeoTabs>
-    <div v-if="!name" class="search-history">
+    <div v-if="!name" class="search-history pt-5 mt-0">
```
Comment (Jarsen136) [inline components/search/SearchSuggestion.vue:173]:
```suggestion
    <div v-if="!name" class="search-history pt-5">
```

**Review by exezbcz — APPROVED** [review]

File: components/search/SearchSuggestion.vue (hunk 1 above)
Comment (floyd-li) [reply components/search/SearchSuggestion.vue:204]: p-4 will change the horizontal padding

File: components/search/SearchSuggestion.vue (hunk 3 above)
Comment (floyd-li) [reply components/search/SearchSuggestion.vue:173]: this `mt-0` is useful

Comment (Jarsen136) [reply components/search/SearchSuggestion.vue:173]: looks like it dose not work as you have deleted the margin from `search-history`

Comment (floyd-li) [reply components/search/SearchSuggestion.vue]: aya yes it is

**Review by Jarsen136 — APPROVED** [review]

**Comment by vikiival** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 4.25 USD/DOT ~ 2.353 $DOT
🧗 [13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A](https://polkadot.subscan.io/account/13rv1SWoLg9Gb3tmvHPZxb7JbVy51BtMziX7k9WQGSJ7Kp3A)
🔗 [0x75649f02978da7c137785be2c1d211ffc3e0d82f5e855c515cef392680f028de](https://polkadot.subscan.io/extrinsic/0x75649f02978da7c137785be2c1d211ffc3e0d82f5e855c515cef392680f028de)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**